roles/openldap-server/templates/override.conf.j2

```diff
@@ -2,4 +2,4 @@
 ExecStartPre=/bin/mkdir -p /run/slapd
 ExecStartPre=/bin/chown openldap:openldap /run/slapd
 ExecStart=
-ExecStart=/usr/sbin/slapd -u {{ openldap_server_user }} -g {{ openldap_server_user }} -h "ldap://127.0.0.1 ldaps:///"
+ExecStart=/usr/sbin/slapd -u {{ openldap_server_user }} -g {{ openldap_server_user }} -h "ldap://{{ openldap_server }} ldaps:///"
```

Comment:
we should not expose the ldap server (should stay on localhost - like the ipv6 instance)